### PR TITLE
refine README after review

### PR DESCRIPTION
**Pull request by zyxiaoyuer**

### CODE REVIEW

File: faq-plus/README.md [hunk 1]
```diff
@@ -35,12 +35,12 @@ FAQ Plus bot is a friendly Q&A bot that brings a human in the loop when it is un
...
+2. Open project in [Visual Studio Code](https://code.visualstudio.com/), ensure you have installed [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit)
+3. Open the command palette by pressing `(Ctrl+Shift+P / ⌘⇧-P or View -> Command - Palette)` and select: `Teams: Provision in the Cloud`. If you have not logged in with your Azure Account, you will be asked to login and select an subscription to provision resources for this Bot.
+4. Once provision is completed, open `.fx/env.default.json` file. Notice the value of `siteName` in `fx-resource-bot` section, this is the name of Azure Web App service provisioned by Teams Toolkit. You will need this value in step 5.
+5. From [Azure Portal](https://ms.portal.azure.com/#home), navigate to resource group named `FaqPlus-rg`. This resource group is created by Teams Toolkit during provision. Find the Azure Web App service named after the value of 'siteName' parameter you got from step 4 and open this Azure Web App service.
```
Comment (dilin-MS) [inline faq-plus/README.md:42]: Use `` (instead of '') for siteName to align with with step 4.

File: faq-plus/README.md [hunk 2]
```diff
@@ -49,7 +49,8 @@ FAQ Plus bot is a friendly Q&A bot that brings a human in the loop when it is un
     | QNAMAKERHOSTURL | `QnAMakerHostUrl` got in [deploy-qna-maker-services](docs/deploy-qna-maker-services.md) step | Example: https://[BaseResourceName]-qnamaker.azurewebsites.net |
     | QNAMAKERSUBSCRIPTIONKEY | `QnAMakerSubscriptionKey` got in [deploy-qna-maker-services](docs/deploy-qna-maker-services.md) step | Usually a 32 alphanumeric characters string |
 7. Open the command palette and select: `Teams: Deploy to the Cloud`.
-8. Once deployment is completed, you can preview the app running in Azure. In Visual Studio Code, open `Run and Debug` and select `Launch Remote (Edge)` or `Launch Remote (Chrome)` in the dropdown list. Then press `F5` or click green arrow button to open the app in Teams web client.
+8. To install and run the app, in Visual Studio Code, open `Run and Debug` tab and select `Launch Remote (Edge)` or `Launch Remote (Chrome)` in the dropdown list.   Then press `F5` or click green arrow button to launch Teams client in browser. Log in to Teams and click `Add` button to install the app to your Teams client.
```
Comment (blackchoey) [inline faq-plus/README.md:52]: Extra spaces before `Then press F5 or click green arrow`

File: faq-plus/README.md [hunk 3]
```diff
@@ -62,41 +63,30 @@ FAQ Plus bot is a friendly Q&A bot that brings a human in the loop when it is un
...
-1. Add the bot to personal chat, then add the bot to a Teams Channel (references as experts channel below). The Teams Channel must under the Team configured in [this](docs/deploy-qna-maker-services.md) doc.
-2. "Ask an expert" in personal chat, you will see the support ticket is posted to experts channel.
-3. Click "Chat with xxx" to chat with the user who asked for help. Note: if the support ticket is created by yourself, you cannot use this feature to chat with yourself.
-4. Click "Change status" button to change the status of the support ticket. The user who created the ticket will receive status change notification from the bot.
-
-## Legal Notice
-
-This app template is provided under the [MIT License](https://github.com/OfficeDev/TeamsFx-Samples/blob/main/LICENSE) terms.  In addition to these terms, by using this app template you agree to the following:
+3. In Visual Studio Code, open `Run and Debug` tab and select `Debug (Edge)` or `Debug (Chrome)` in the dropdown list. Then press `F5` or click green arrow button to start local debug and open the app in Teams web client.
```
Comment (dilin-MS) [inline faq-plus/README.md:66]: Do we need to add "Your app is successfully running in your Teams client now, refer to ['Use the App' section](./README#use-the-app) below to interact with the app.“ here since local debug needs to install the app twice, too.

File: faq-plus/README.md [hunk 4]
```diff
@@ -49,7 +49,8 @@ FAQ Plus bot is a friendly Q&A bot that brings a human in the loop when it is un
     | QNAMAKERHOSTURL | `QnAMakerHostUrl` got in [deploy-qna-maker-services](docs/deploy-qna-maker-services.md) step | Example: https://[BaseResourceName]-qnamaker.azurewebsites.net |
     | QNAMAKERSUBSCRIPTIONKEY | `QnAMakerSubscriptionKey` got in [deploy-qna-maker-services](docs/deploy-qna-maker-services.md) step | Usually a 32 alphanumeric characters string |
 7. Open the command palette and select: `Teams: Deploy to the Cloud`.
-8. Once deployment is completed, you can preview the app running in Azure. In Visual Studio Code, open `Run and Debug` and select `Launch Remote (Edge)` or `Launch Remote (Chrome)` in the dropdown list. Then press `F5` or click green arrow button to open the app in Teams web client.
+8. To install and run the app, in Visual Studio Code, open `Run and Debug` tab and select `Launch Remote (Edge)` or `Launch Remote (Chrome)` in the dropdown list.   Then press `F5` or click green arrow button to launch Teams client in browser. Log in to Teams and click `Add` button to install the app to your Teams client.
+9. Your app is successfully running in your Teams client now, refer to ['Use the App' section](./README#use-the-app) below to interact with the app.
```
Comment (dilin-MS) [inline faq-plus/README.md:53]: Seems the anchor should be ['Use the App' section](#use-the-app). Could you help confirm that?

File: faq-plus/README.md [hunk 5]
```diff
@@ -49,7 +49,8 @@ FAQ Plus bot is a friendly Q&A bot that brings a human in the loop when it is un
...
 
 ### (Optional) Debug
```
Comment (dilin-MS) [inline faq-plus/README.md:55]: Why debug is optional? Debug is supposed be an important development step in the inner loop.

File: faq-plus/README.md [hunk 6]
```diff
@@ -49,7 +49,8 @@ FAQ Plus bot is a friendly Q&A bot that brings a human in the loop when it is un
...
```
Comment (dilin-MS) [inline faq-plus/README.md:56]: Better to keep wording in the first step just the same as the first step in #try-the-sample section to avoid confusion since they do exactly the same thing.

File: faq-plus/docs/deploy-qna-maker-services.md [hunk 7]
```diff
@@ -37,23 +37,20 @@ We recommend that you copy these values into a text file, using an application l
 
 1. Click on the "Deploy to Azure" button below.
 
-[![Deploy to Azure](https://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FOfficeDev%2FTeamsFx-Samples%2Fdev%2Ffaq-plus%2Fdeployment%2Fazuredeploy.json)
+[![Deploy to Azure](https://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FOfficeDev%2FTeamsFx-Samples%2Ffeature%2Ffaq-plus%2Ffaq-plus%2Fdeployment%2Fazuredeploy.json)
```
Comment (blackchoey) [inline faq-plus/docs/deploy-qna-maker-services.md:40]: please help revert this change, the url changes in dev branch

File: faq-plus/README.md (hunk 5 above)
Comment (zyxiaoyuer) [reply faq-plus/README.md:55]: sample app is complete and tested, not developed indepently, debugging is optional when user want to built their own on sample . But befofer that, they want to deploy and run the app first

File: faq-plus/README.md (hunk 6 above)
Comment (zyxiaoyuer) [reply faq-plus/README.md:56]: done

**Review by blackchoey — APPROVED** [review]
